go/cmd/dolt/commands/clone.go

```diff
@@ -231,10 +241,11 @@ func cloneAllBranchRefs(branches []ref.DoltRef, srcDB *doltdb.DoltDB, ctx contex
 
 		progChan := make(chan datas.PullProgress)
 		doneChan := make(chan struct{})
-		go progFunc(progChan, doneChan)
+		go progFunc(progChan, doneChan, evt)
 
```

Comment:
So I modified `progFunc` , in order to get the bytes transferred and the duration into the metrics and register the metrics into the `evt`, but I'm not sure this is the right move...? I think I need to dig a bit deeper to find the exact byte count and not reference the approximation for metrics?